index.html

```diff
@@ -5,7 +5,10 @@
 
     <meta charset="utf-8"/>
     <meta http-equiv="x-ua-compatible" content="ie=edge"/>
-    <meta name="viewport" content="width=device-width"/>
+	<meta name="viewport" content="width=device-width"/>
```

Comment:
dlaczego zmieniamy wcięcie?

Reply:
Przeoczenie, VSCode po clonie tego repo nadal nie pokazuje niewłaściwych wcięć. Za to przeczesanie `prettierem`  i push daje oczekiwany już rezultat.